site/content/docs/main/how-velero-works.md

```diff
@@ -24,7 +24,7 @@ Note that cluster backups are not strictly atomic. If Kubernetes objects are bei
 
 ## Scheduled backups
 
-The **schedule** operation allows you to back up your data at recurring intervals. The first backup is performed when the schedule is first created, and subsequent backups happen at the schedule's specified interval. These intervals are specified by a Cron expression.
+The **schedule** operation allows you to back up your data at recurring intervals. You can create a scheduled backup at any time, and the first backup is then performed at schedule's specified interval. These intervals are specified by a Cron expression.
```

Comment:
at schedule's -> at the schedule's

Add link to https://en.wikipedia.org/wiki/Cron

Reply:
update the grammer change, but didn't include the link. There is a link to the wiki page on the schedule backup reference page. I feel like crons are common enough that maybe we dont need the link here